gbpservice/nfp/lib/transport.py

```diff
@@ -174,27 +174,43 @@ def send_request_to_configurator(conf, context, body,
     # method (tcp_rest/rpc) for sending request to configurator.
     if device_config:
         method_name = method_type.lower() + '_network_function_device_config'
-        for ele in body['config']:
-            ele['kwargs'].update({'context': context.to_dict()})
+        body['info']['context'].update({'neutron_context': context.to_dict()})
```

Comment:
why we are sending neutron_context inside context, it is not specified in blueprint.